src/reducers/slices/gasPrice.js

```diff
@@ -0,0 +1,15 @@
+import { createReducer } from '../util';
+import * as types from '../../constants/actionTypes';
+
+const initialSettings = {};
+
+const loadingGasPrice = () => null;
+
+const loadedGasPrice = (gas, { payload }) => payload.gasPrice;
+
+const gasPriceReducer = createReducer(initialSettings, {
```

Comment:
Looks like either initialSettings should be a number, or the gasPrice should be stored as a property of the state object

Reply:
I've updated the initialSettings to be a number instead.